yb-voyager/cmd/exportData.go

```diff
@@ -766,11 +785,23 @@ func getFinalTableColumnList() (map[string]string, []sqlname.NameTuple, *utils.S
 			}
 		})
 	}
+	var partitionsToRootTableMap map[string]string
+	isTableListSet := source.TableList != ""
+	partitionsToRootTableMap, finalTableList, err = addLeafPartitionsInTableList(finalTableList, isTableListSet)
+	if err != nil {
+		utils.ErrExit("failed to add the leaf partitions in table list: %w", err)
+	}
+
+	return partitionsToRootTableMap, finalTableList, isTableListSet
+}
+
+func finalizeTableColumnList(finalTableList []sqlname.NameTuple, isTableListSet bool) ([]sqlname.NameTuple, *utils.StructMap[sqlname.NameTuple, []string]) {
```

Comment:
is `isTableListSet` is required in finalizeTableColumnList ? if not remove the parameter

Reply:
Done